### DISCUSSION TITLE
how to get LazyVim with pyright to insert parentheses on autocomplete for methods

### DISCUSSION THREAD

**Discussion by adam006** [LazyVim/LazyVim#2868] [General]
I've been trying out some of the different neovim distros and really have enjoyed LazyVim. One of the things I've found that is different though is that NvChad and LunarVim with pyright will insert the `()` on autocomplete for methods but LazyVim does not for some reason. Does anyone have any experience with this and could help me get this functionality in LazyVim?

**Comment by dpetka2001** [comment]
You need to setup [nvim-autopairs](https://github.com/windwp/nvim-autopairs) to have the same behavior you observe in other distros. That is my assumption at least.

**Comment by folke** [comment]
Just added support for this in LazyVim (without nvim-autopairs):

See:
- https://github.com/LazyVim/LazyVim/commit/bf8ce8076e283ecabce6b2f2a04995b3deeeb593
- https://github.com/LazyVim/LazyVim/commit/d95c2ba1dc75307b154c870aa808f322debc6070
